src/theory/tools/lifter/bir_riscv_extrasScript.sml

```diff
@@ -283,7 +287,7 @@ let
   ];
 
   val elim_zero_thm =
-    GEN_ALL (SIMP_CONV (std_ss++wordsLib.WORD_ss) [] ``riscv_mem_store_byte (a+0w) w mmap``);
+    GEN_ALL (SIMP_CONV (std_ss++wordsLib.WORD_ss) [] ``mem_store_byte (a+0w) w mmap``);
```

Comment:
I thought the `riscv_mem_` functions were necessary? Is there an obvious reason for why no longer needed here?

Reply:
This occurrence makes more sense if you look at what's going on just above: rewriting of the `riscv_mem` functions in terms of `mem_store_byte`. The usage of `riscv_mem_store_byte` was probably an earlier typo.

I did some tests to understand what's going on here, and I think the current usage of `mem_store_byte` should stay.